src/components/modals/VoteHistoryModal.vue

```diff
@@ -1,17 +1,43 @@
 <template>
   <Modal
     class="vote-history"
-    v-if="modals.voteHistory && session.voteHistory"
+    v-if="modals.voteHistory && (session.voteHistory || !session.isSpectator)"
     @close="toggleModal('voteHistory')"
   >
     <font-awesome-icon
       @click="clearVoteHistory"
       icon="trash-alt"
       class="clear"
       title="Clear history"
+      v-if="session.isSpectator"
     />
 
     <h3>Nomination history</h3>
+
+    <template v-if="!session.isSpectator">
+      <table>
```

Comment:
This isn't a table and shouldn't be written as one. Just use `<div class="options"><div>option</div><div>option</div></div>` and let CSS (and flexbox) handle the rest. If you need an example for this, take a look at `.button-group`